packages/utilities/src/dom/getTopMostDocument.ts

```diff
@@ -0,0 +1,13 @@
+/**
+ * Helper to get the top-most document object.
+ *
+ * @public
+ */
+export function getTopMostDocument(): Document | undefined {
```

Comment:
I'm wondering if other names would be simpler and just as clear, like `getRootDocument`, `getTopDocument`, etc.

Reply:
`getRootDocument` sounds good, I'll update it.

Reply:
See bdeeba7dfb672fd4b2db840b6d065a8a629d3953